src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -386,6 +391,11 @@
           "type": "string"
         },
 
+        "decoratedName": {
+          "description": "The fully qualified name for the logical location, in the compiler's native format.",
```

Comment:
'The non-readable fully qualified name of the location where the analysis tool produced the result, such as the mangled function name provided by a C++ compiler that encodes calling convention, return type and other details along with the function name.'

Also suggest adding the qualifier 'readable' to the 'fullyQualifiedLogicalName' description. What do you think?